hack/lib/cleanup.sh

```diff
@@ -152,4 +152,18 @@ function os::cleanup::tmpdir() {
 		${USE_SUDO:+sudo} rm -rf "${directory}"
 	done
 }
-readonly -f os::cleanup::tmpdir
\ No newline at end of file
+readonly -f os::cleanup::tmpdir
+
+# os::cleanup::dump_events dumps all the events from a cluster to a file.
+#
+# Globals:
+#  ARTIFACT_DIR
+# Arguments:
+#  None
+# Returns:
+#  None
+function os::cleanup::dump_events() {
+	os::log::info "Dumping cluster events to ${ARTIFACT_DIR}/events.txt"
+	oc get events --all-namespaces > "${ARTIFACT_DIR}/events.txt"
```

Comment:
does this require admin? could use `--config="${ADMIN_KUBECONFIG}"` if that's set